ebuild-writing/functions/src_test/text.xml

```diff
@@ -114,26 +107,15 @@ suite is extremely helpful for arch maintainers.
 <title>Skipping Tests</title>
 <body>
 <p>
-Sometimes it is necessary to skip tests entirely. This can be done
-using a dummy <c>src_test</c> function:
-</p>
-
-<codesample lang="ebuild">
-src_test() {
-    # Tests don't even remotely work inside portage
-    true
-}
-</codesample>
-
-<p>
-Another option would be to set <c>RESTRICT="test"</c> in the ebuild.
+Sometimes it is necessary to skip tests entirely. This can be done by
+setting <c>RESTRICT="test"</c> in the ebuild.
 </p>
 
 <note>
 If upstream provide a test suite which doesn't work, consider talking
-to them about getting it fixed. A broken test suite is worse than no
-test suite at all, since we are unable to tell whether a test failure
-indicates a genuine fault.
+to them about getting it fixed. A broken test suite requires developers
+to investigate each test failure in order to determine whether it is
+genuine or indicates a broken test.
```

Comment:
I personally like the old version better. Likewise a broken test suite, a non-broken test suite also requires developers to investigate each test failure since there's no way `to tell whether a test failure indicates a genuine fault.`